mergify_engine/subscription_key.py

```diff
@@ -0,0 +1,182 @@
+# -*- encoding: utf-8 -*-
```

Comment:
I'm not sure about this as it's a pain to update the file then.
I'd rather start with a simple mode where the engine call a public API with a secret to get a JSON description of its subscription.